core/client/app/templates/setup/two.hbs

```diff
@@ -18,22 +18,24 @@
             </div>
             <a class="edit-account-image" href="#"><i class="icon-photos"><span class="sr-only">Upload an image</span></i></a>
         </figure>
-        <div class="form-group">
+        {{#gh-form-group errors=errors property="email"}}
             <label for="email-address">Email address</label>
             <span class="input-icon icon-mail">
-                {{input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email}}
+                {{gh-input type="email" name="email" placeholder="Eg. john@example.com" class="gh-input" autofocus="autofocus" autocorrect="off" value=email focusOut=(action "validate" "email")}}
             </span>
-        </div>
-        <div class="form-group">
+            {{gh-error-message errors=errors property="email"}}
+        {{/gh-form-group}}
+        {{#gh-form-group errors=errors property="name"}}
             <label for="full-name">Full name</label>
             <span class="input-icon icon-user">
-                {{input type="text" name="name" placeholder="Eg. John H. Watson" class="gh-input" autofocus="autofocus" autocorrect="off" value=name }}
+                {{gh-input type="text" name="name" placeholder="Eg. John H. Watson" class="gh-input" autofocus="autofocus" autocorrect="off" value=name focusOut=(action "validate" "name")}}
             </span>
-        </div>
-        <div class="form-group">
+            {{gh-error-message errors=errors property="name"}}
+        {{/gh-form-group}}
+        {{#gh-form-group errors=errors property="password"}}
             <label for="password">Password</label>
             <span class="input-icon icon-lock">
-                {{input type="password" name="password" placeholder="At least 8 characters" class="gh-input" autofocus="autofocus" autocorrect="off" value=password }}
+                {{gh-input type="password" name="password" placeholder="At least 7 characters" class="gh-input" autofocus="autofocus" autocorrect="off" value=password focusOut=(action "validate" "password")}}
```

Comment:
The placeholder text has an incorrect password length again (probably a rebase/merge issue).